### PR TITLE
I18n fix

### PR DESCRIPTION
**Pull request by sgiaccio**

### CODE REVIEW

File: server/modules/taxonomy/service/taxonomyImportJob.js [hunk 1]
```diff
@@ -166,7 +166,10 @@ class TaxonomyImportJob extends Job {
       const duplicateCodeRow = this.codesToRow[code]
 
       if (duplicateCodeRow) {
-        validation = Validator.assocFieldValidation(Taxon.propKeys.code, { valid: false, errors: [{ key: Validator.errorKeys.duplicate }] })(validation)
+        validation = Validator.assocFieldValidation(Taxon.propKeys.code, {
+          valid: false,
+          errors: [{ key: 'duplicateCode', params: { row: duplicateCodeRow, duplicateRow: this.processed + 1 } }],
```
Comment (minotogna) [inline server/modules/taxonomy/service/taxonomyImportJob.js]: should we add duplicateCode to ```Validator.errorKeys```?

File: server/modules/taxonomy/service/taxonomyImportJob.js [hunk 2]
```diff
@@ -175,15 +178,18 @@ class TaxonomyImportJob extends Job {
       const duplicateScientificNameRow = this.scientificNamesToRow[scientificName]
 
       if (duplicateScientificNameRow) {
-        validation = Validator.assocFieldValidation(Taxon.propKeys.scientificName, { valid: false, errors: [{ key: Validator.errorKeys.duplicate }] })(validation)
+        validation = Validator.assocFieldValidation(Taxon.propKeys.scientificName, {
+          valid: false,
+          errors: [{ key: 'duplicateName', params: { row: duplicateCodeRow, duplicateRow: this.processed + 1 } }],
```
Comment (minotogna) [inline server/modules/taxonomy/service/taxonomyImportJob.js]: should we add duplicateName to ```Validator.errorKeys```?

File: webapp/commonComponents/form/checkbox.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ class Checkbox extends React.Component {
 
     return (
       <div style={{ justifySelf: 'start' }}>
-        <ValidationTooltip validation={validation}>
+        <ValidationTooltip validation={validation} showKeys={false}>
```
Comment (minotogna) [inline webapp/commonComponents/form/checkbox.js]: checkbox is generic. should showKeys be passed from the caller?

File: webapp/commonComponents/form/input.js [hunk 4]
```diff
@@ -26,6 +26,7 @@ export const Input = React.forwardRef((props, ref) => {
   return (
     <ValidationTooltip
       validation={validation}
+      showKeys={false}
```
Comment (minotogna) [inline webapp/commonComponents/form/input.js]: input is generic. should showKeys be passed from the caller?

File: webapp/loggedin/surveyViews/surveyForm/nodeDefs/nodeDefSwitch.js [hunk 5]
```diff
@@ -82,7 +82,8 @@ class NodeDefSwitch extends React.Component {
                            edit={edit}
                            parentNode={parentNode}
                            nodes={nodes}
-                           container={this.element}/>
+                           container={this.element}
+                           showKeys={false}/>
```
Comment (minotogna) [inline webapp/loggedin/surveyViews/surveyForm/nodeDefs/nodeDefSwitch.js]: is NodeDefErrorBadge showKeys param always false?

File: server/modules/taxonomy/service/taxonomyImportJob.js (hunk 2 above)
Comment (sgiaccio) [reply server/modules/taxonomy/service/taxonomyImportJob.js]: Ok

File: server/modules/taxonomy/service/taxonomyImportJob.js (hunk 1 above)
Comment (sgiaccio) [reply server/modules/taxonomy/service/taxonomyImportJob.js]: Ok

File: webapp/commonComponents/form/checkbox.js (hunk 3 above)
Comment (sgiaccio) [reply webapp/commonComponents/form/checkbox.js]: In fact showKeys is not needed in this case, as there's no 'field' attribute in the validation object, I removed showKeys in <validationTooltip />

File: webapp/commonComponents/form/input.js (hunk 4 above)
Comment (sgiaccio) [reply webapp/commonComponents/form/input.js]: In fact showKeys is not needed in this case, as there's no 'field' attribute in the validation object, I removed showKeys in <validationTooltip />

File: webapp/loggedin/surveyViews/surveyForm/nodeDefs/nodeDefSwitch.js (hunk 5 above)
Comment (sgiaccio) [reply webapp/loggedin/surveyViews/surveyForm/nodeDefs/nodeDefSwitch.js]: showKeys is not passed in nodeDefEntityForm, so in that case it's using the default value (true)

Comment (minotogna) [reply webapp/loggedin/surveyViews/surveyForm/nodeDefs/nodeDefSwitch.js]: but should we always hide the keys in NodeDefEntityForm as well? what do you think?

**Pull request merged**